Makefile

```diff
@@ -16,6 +16,7 @@ help:
 	@echo "  start-compose            to compose all containers in detached state"
 	@echo "  stop-compose             to stop all containers"
 	@echo "  start-db                 to start the psql db in detached state"
+	@echo "  docs-seqdiag             to regenerate docs .svg files from .diag files"
```

Comment:
Would you mind making an overarching `make docs` that would call `docs-seqdiag` as part of it? That way we have a simple command to call that would generate _all_ docs (as soon as we have them 😄)

Reply:
Now we have `made doc` and `make doc-seqdiag`.

For context, we couldn't call it `make docs` because make doesn't like having a target be the same name as a local directory name. 🤦‍♂️ https://stackoverflow.com/a/4435491/6621995